docs/UserGuide.md

```diff
@@ -945,21 +890,32 @@ Format: `exit`
 
 On the profile page,<br>
 
-* Entering <code>exit</code> leads you out of the page back to the main window. 
-* Attempting to <code>exit</code> **without first saving changes** with <code>save</code> prompts a warning message (please refer to [User Interface: The Profile Page: Exiting the profile page](#exiting-the-profile-page) for more information). 
+* Entering <code>exit</code> leads you back to the main window. 
+* Attempting to <code>exit</code> **without first saving changes** with <code>save</code> prompts a warning message.
```

Comment:
same for this, "without saving changes", I think it sounds more natural without a "first" there